pkg/tasks/tasks.go

```diff
@@ -352,6 +352,11 @@ func WithUpgrade(t Tasks, followers ...kubeoneapi.HostConfig) Tasks {
 				Description: "upgrade MachineDeployments",
 				Predicate:   func(s *state.State) bool { return s.UpgradeMachineDeployments },
 			},
+			Task{
+				Fn: pruneImagesOnAllNodes, Operation: "pruning unused container images",
+				Description: "prune unused container images",
+				Predicate:   func(s *state.State) bool { return s.PruneImages },
```

Comment:
Nit: `Fn` and `Operation` should be on two lines, also let's replace `pruning` with `deleting` to standardize the naming.